backend/uclapi/.env.docker.example

```diff
@@ -35,7 +37,7 @@ WHITELISTED_CALLBACK_URLS=https://live-roombookings.uclapi.com/webhook/;https://
 DB_UCLAPI_NAME=uclapi
 DB_UCLAPI_USERNAME=uclapi
 DB_UCLAPI_PASSWORD=uclapi_test_password
-DB_UCLAPI_HOST=localhost
+DB_UCLAPI_HOST=db
```

Comment:
`.env.docker.example` is designed to be used with the `docker-compose-dev.yml` which exposes the database, gencache etc. to the host. These DNS names won't be able to be resolved (unless using the uclapi docker container).